src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/HerculesPickaxe.java

```diff
@@ -32,18 +32,12 @@ public HerculesPickaxe(ItemGroup itemGroup, SlimefunItemStack item, RecipeType r
             if (SlimefunTag.ORES.isTagged(mat)) {
                 if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_17)) {
                     switch (mat) {
-                        case DEEPSLATE_IRON_ORE:
-                            drops.add(new CustomItemStack(SlimefunItems.IRON_DUST, 2));
-                            break;
-                        case DEEPSLATE_GOLD_ORE:
-                            drops.add(new CustomItemStack(SlimefunItems.GOLD_DUST, 2));
-                            break;
-                        case COPPER_ORE:
-                        case DEEPSLATE_COPPER_ORE:
-                            drops.add(new CustomItemStack(SlimefunItems.COPPER_DUST, 2));
-                            break;
-                        default:
-                            break;
+                        case DEEPSLATE_IRON_ORE -> drops.add(new CustomItemStack(SlimefunItems.IRON_DUST, 2));
+                        case DEEPSLATE_GOLD_ORE -> drops.add(new CustomItemStack(SlimefunItems.GOLD_DUST, 2));
+                        case COPPER_ORE, DEEPSLATE_COPPER_ORE ->
+                                drops.add(new CustomItemStack(SlimefunItems.COPPER_DUST, 2));
+                        default -> {
+                        }
                     }
```

Comment:
There is another (almost equivalent) switch/case a few lines down.
Why was this updated and the one below not?

Reply:
Missed it when doing the auto-update using my IDE. Fixed now